Bendy and the Dark Revival/bendyandthedarkrevival.asl

```diff
@@ -143,7 +157,7 @@ split
         return true;
     }
 
-    if (vars.Setting("la") && current.cutsceneID == 11707 && current.LordAmok == "Amok Follower: Hail the New Amok!" && old.LordAmok != "Amok Follower: Hail the New Amok!")
+    if (current.cutsceneID == 11707 && current.LordAmok == "Amok Follower: Hail the New Amok!" && old.LordAmok != "Amok Follower: Hail the New Amok!" && vars.Setting("la", true))
```

Comment:
it would be nicer to keep the vars.Setting check first so that we can early exit this check if someone doesn't have this setting enabled or it was already completed

in reality it's minor but we should keep this convention I feel